spec/features/navigation_bar_spec.rb

```diff
@@ -1,17 +1,23 @@
 require 'rails_helper'
 feature "Navigation bar" do 
-  scenario "has link sign in and home link when user is not signed on" do 
+  before(:example) do |example|
+    unless example.metadata[:skip_before]
+      @user = create(:user)
+      manual_sign_in(@user)
+      visit root_path
+    end
+  end
+
+  scenario "has sign in link, sign up link, and home link when user is not signed in", :skip_before do 
```

Comment:
This description is getting long. Descriptions don't need to contain every detail, that's what the code inside the test is for. The description can be a summary of what the test is doing. For example, here you could write:

```ruby
scenario "has appropriate links for anonymous users", :skip_before do
```